blaze/compute/tests/test_sql_compute.py

```diff
@@ -1,22 +1,51 @@
 from __future__ import absolute_import, division, print_function
 
+import re
+
 import pytest
 sqlalchemy = pytest.importorskip('sqlalchemy')
 sa = sqlalchemy
 
-import re
-
 import datashape
+from odo import into, resource
+from pandas import DataFrame
+from toolz import unique
 
 from blaze.compute.sql import (compute, computefull, select, lower_column,
                                compute_up)
-from blaze.expr import *
+from blaze.expr import (symbol, discover, transform, summary, by, sin, join,
+                        floor, cos, merge, nunique, distinct, mean, sum,
+                        count, exp, nelements)
 from blaze.compatibility import xfail
-from toolz import unique
-from pandas import DataFrame
-from odo import into, resource
 from blaze.utils import tmpfile
 
+
+@pytest.fixture(scope='module')
+def data():
```

Comment:
No table inserts or changes that I know of. Maybe I should just switch it now since it might catch someone off guard in the future?

I don't think we're ready to add the *yield* fixture yet, since this just uses an sqlite db.